src/foam/nanos/auth/UserAndGroupAuthService.js

```diff
@@ -0,0 +1,440 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'UserAndGroupAuthService',
+  flags: ['java'],
+
+  implements: [
```

Comment:
The current `UserAndGroupAuthService` extends `ContextAwareSupport` but this doesn't.

Reply:
All FObjects are context aware. Redundant code. Hence, removed.